sessionctx/variable/varsutil.go

```diff
@@ -535,3 +536,14 @@ var GAFunction4ExpressionIndex = map[string]struct{}{
 	ast.VitessHash: {},
 	ast.TiDBShard:  {},
 }
+
+func getRuntimeTLSVersion() string {
+	tlsVersion := config.GetGlobalConfig().Security.MinTLSVersion
+	for _, v := range tlsVersionString {
+		if v == tlsVersion {
+			return tlsVersion
+		}
+	}
+	// default
+	return "TLSv1.1"
+}
```

Comment:
https://docs.pingcap.com/tidb/dev/tidb-configuration-file#tls-version `MinTLSVersion` specifies the lower bound not the exact version. And the default enabled version should be "TLSv1.1,TLSv1.2,TLSv1.3".

Reply:
https://github.com/pingcap/tidb/blob/441c9cde3053ed23a8809e306c5aaf74d61b95e2/util/misc.go#L470-L492
But we discuss yesterday, it should be the final result after apply the configuartion. I find if we not configure it, It will be `VersionTLS11`.

Reply:
Maybe I don't make myself clear. After apply the configuration, for example if we configure tls-version to TLSv1.2, then the value should be TLSv1.2,TLSv1.3.

Reply:
Thanks, i get it. I will fix it later